### ISSUE TITLE
Custom Camera Card Only Shows Up Once After Login

### ISSUE THREAD

**Issue by StarScream159** [UI-Lovelace-Minimalist/UI#1452]
**Describe the bug**
I am using the [custom_card_camera](https://ui-lovelace-minimalist.github.io/UI/usage/custom_cards/custom_card_camera/) card, however it only shows the camera image once after logging in. If I refresh my browser, or close the tab and reopen home assistant, the card disappears. This happens to all 4 of my camera cards on any page or dashboard I have them setup in.

**To Reproduce**

1. Add the custom card to your yaml:
```yaml
          - type: "custom:button-card"
            entity: "camera.blink_backyard_camera"
            template: "custom_card_camera"
            show_icon: true #True if you set ulm_custom_card_camera_title: true
            variables:
              ulm_custom_card_camera_title: true
              ulm_custom_card_camera_name: "Backyard"
              ulm_custom_card_camera_label: "Camera"
```
2. Log out of home assistant
3. Log back in and see the camera card without issue
4. Reload the browser (F5)
5. The cards are no longer rendered

**Expected behavior**
I would expect the camera card to show each time.

**Additional context**
I tried looking at the console for any errors but wasn't able to find any that relate to the camera card. I see a bunch of errors in general, but they seem to always be there even when the camera card works as expected.

Core 2024.1.6
Supervisor 2023.12.1
Operating System 11.4
Frontend 20240104.0
UI Lovelace Minimalist v1.3.9

Frontend Additions (from HACS):
- apexcharts-card
- Kiosk Mode
- Dual gauge card

**Related Card Config**
```yaml
      - type: 'custom:button-card'
        template: card_title
        name: Security
      - type: grid
        columns: 2
        square: false
        cards:
          - type: "custom:button-card"
            entity: "camera.blink_front_door_2"
            template: "custom_card_camera"
            show_icon: true #True if you set ulm_custom_card_camera_title: true
            variables:
              ulm_custom_card_camera_title: true
              ulm_custom_card_camera_name: "Front"
              ulm_custom_card_camera_label: "Doorbell"
          - type: "custom:button-card"
            entity: "camera.blink_front_camera"
            template: "custom_card_camera"
            show_icon: true #True if you set ulm_custom_card_camera_title: true
            variables:
              ulm_custom_card_camera_title: true
              ulm_custom_card_camera_name: "Front"
              ulm_custom_card_camera_label: "Camera"
          - type: "custom:button-card"
            entity: "camera.blink_side_camera"
            template: "custom_card_camera"
            show_icon: true #True if you set ulm_custom_card_camera_title: true
            variables:
              ulm_custom_card_camera_title: true
              ulm_custom_card_camera_name: "Side"
              ulm_custom_card_camera_label: "Camera"
          - type: "custom:button-card"
            entity: "camera.blink_backyard_camera"
            template: "custom_card_camera"
            show_icon: true #True if you set ulm_custom_card_camera_title: true
            variables:
              ulm_custom_card_camera_title: true
              ulm_custom_card_camera_name: "Backyard"
              ulm_custom_card_camera_label: "Camera"
```

**Comment by enricofranke** [comment]
I can confirm that I have the same problem.

After rebooting HA, once the camera card is loaded after a page reload, the page card is gone and never comes back.

**Comment by StarScream159** [comment]
This issue stopped for me after having to manually add all the library resources to my install. I had to do this to fix issue #1454 and once completed the camera cards started working properly even after reloading the page or closing my browser.

So to be clear, uncheck the option to load resources: Settings -> Device s& Integrations -> UI Lovelace Minimalist -> Configure -> Uncheck "include custom card resources it's depending on"

Then follow the instructions in the readme to install all the required card resources and load their JavaScript libraries. HACS makes this really easy.
